src/plugins/dmt-app/src/index.tsx

```diff
@@ -2,44 +2,37 @@ import * as React from 'react'
 import { useContext } from 'react'
 import {
   ApplicationContext,
-  EPluginType,
-  Header,
   FSTreeProvider,
+  IUIPlugin,
+  TPlugin,
 } from '@development-framework/dm-core'
 import SearchPage from './pages/SearchPage'
 import ViewPage from './pages/ViewPage'
 import { Route } from 'react-router-dom'
-import Editor from './pages/editor/Editor'
-import { createGlobalStyle, ThemeProvider } from 'styled-components'
+import { ThemeProvider } from 'styled-components'
+import { plugins as headerPlugins } from '@development-framework/header'
 
-const GlobalStyle = createGlobalStyle`
-  body {
-    padding: 0;
-    margin: 0;
-    font-family: Equinor-Regular, sans-serif;
-    -webkit-font-smoothing: antialiased;
-    -moz-osx-font-smoothing: grayscale;
-  }
-`
 const theme = {
   flexboxgrid: {
     gutterWidth: 0, // rem
     outerMargin: 0, // rem
   },
 }
 
-const PageComponent = () => {
+const DmtApp = (props: IUIPlugin) => {
+  const { idReference, config, type } = props
   const applicationSettings = useContext(ApplicationContext)
+  const headerPlugin: TPlugin = headerPlugins.find((plugin) => {
+    return plugin.pluginName === 'header'
+  }) || {
```

Comment:
```suggestion
  }) ?? {
```
Is this a use-case for the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing)?